samples/keys/identity_key_usage/README.rst

```diff
@@ -0,0 +1,81 @@
+.. _identity_key_usage:
+
+Identity Key Usage
+##################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample shows how to use a previously written identity key with the PSA crypto APIs.
+
+Requirements
+************
+
+The following development kits are supported:
+
+.. table-from-sample-yaml::
+
+The :ref:`lib_hw_unique_key` library is required to generate the prerequisite Master Key Encryption Key (MKEK).
+The :ref:`lib_identity_key` library is required to provision the identity key in KMU.
+Both these operations can be done by running the :ref:`identity_key_generate` sample.
+
+.. note::
+   Erase only the code pages when programming the device after the required identity key has been provisioned on the device, as ERASEALL removes the identity key from the system.
```

Comment:
```suggestion
   Erase the code pages when programming the device only after the required identity key has been provisioned on the device, as ERASEALL removes the identity key from the system.
```

Reply:
Please check if you accept the previous suggestion and I can update both.